code/run_capsule.py

```diff
@@ -203,17 +203,23 @@ def plot_raw_dff_mc(
     parser.add_argument("--no_qc", action="store_true", help="Skip QC plots.")
     args = parser.parse_args()
 
-    behavior_path = glob.glob(args.fiber_path + "/behavior/*.json")[0]
-    # Regular expression to match the desired number
-    match = re.search(r'behavior/(\d{6,7})_', behavior_path)
+    session_json_path = glob.glob(args.fiber_path + "/session.json")[0]
```

Comment:
This might be nit but 2 things. 

If the file path is always **/data/fiber_raw_data + session.json**, I would suggest you do os.path.join("/data/fiber_raw_data/session.json"). Then you would need to check if the file path exists and raise and error to let the user know where the code was looking for the file path. Log this message to the log handler

It looks like Path is used in parts of the code. I would convert paths upfront to the Path object so that all file types are handled in the same way

Reply:
This makes sense, I'll work on this now. Using Path for all these for consistency is smart